shared/src/ledger/storage/mod.rs

```diff
@@ -68,6 +72,16 @@ where
     /// Wrapper txs to be decrypted in the next block proposal
     #[cfg(feature = "ferveo-tpke")]
     pub tx_queue: TxQueue,
+    /// Minimum number of confirmations needed for the
+    /// Ethereum bridge to consider an event final
+    pub min_confirmations: Option<MinimumConfirmations>,
+    /// The Ethereum address of the ERC20 contract that represents this chain's
+    /// native token.
+    pub native_erc20: Option<EthAddress>,
+    /// The Ethereum address of the bridge contract.
+    pub bridge_contract: Option<UpgradeableContract>,
+    /// The Ethereum address of the governance contract.
+    pub governance_contract: Option<UpgradeableContract>,
```

Comment:
Do we have to put these on the `Storage` struct? Can't we store these values similarly to however other values set at genesis are stored (e.g. treasury params, gov params, pos params)